Build fails on workbox injectManifest

Error output after running `npm install && npm run build`:
```
> npx workbox injectManifest workbox-config.js

command not found: workbox
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lia-script@0.5.0 sw: `npx workbox injectManifest workbox-config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lia-script@0.5.0 sw script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

After running `npm install workbox-cli && npm run build`:
```
> npx workbox injectManifest workbox-config.js

Using configuration from /home/jacol/Documents/elm/LiaScript/workbox-config.js.
Service worker generation failed:

Unable to find a place to inject the manifest. Please ensure that your service worker file contains the following: self.__WB_MANIFEST
    at Object.injectManifest (/home/jacol/Documents/elm/LiaScript/node_modules/workbox-build/build/inject-manifest.js:168:11)
    at async runBuildCommand (/home/jacol/Documents/elm/LiaScript/node_modules/workbox-cli/build/app.js:49:9)
    at async module.exports (/home/jacol/Documents/elm/LiaScript/node_modules/workbox-cli/build/app.js:138:11)
```

--- a/package.json
+++ b/package.json
@@ -39,7 +39,8 @@
     "dexie": "^2.0.4",
     "katex": "^0.10.2",
     "material-icons": "^0.3.1",
-    "sass-material-colors": "0.0.5"
+    "sass-material-colors": "0.0.5",
+    "workbox-cli": "^5.0.0"
   },
   "browserslist": [
     "cover 99.5%"

--- a/src/javascript/sw.js
+++ b/src/javascript/sw.js
@@ -21,4 +21,4 @@ workbox.routing.registerRoute(
   new workbox.strategies.StaleWhileRevalidate()
 )
 
-workbox.precaching.precacheAndRoute([]);
+workbox.precaching.precacheAndRoute(self.__WB_MANIFEST);

